app/views/layouts/_navbar.haml

```diff
@@ -27,6 +27,10 @@
         .dropdown-menu{aria: { labelledby: 'management-dropdown' }}
           = link_to 'Users', users_path, class: 'dropdown-item'
           = link_to 'Mobility Devices', mobility_devices_path, class: 'dropdown-item'
+          = link_to 'Carriers', carriers_path, class: 'dropdown-item'
+    - elsif @current_user&.dispatcher?
+      %li.nav-item
```

Comment:
Let's show the "Manage" dropdown for everyone, and just show different links within to various people.